src/main/kotlin/pt/up/fe/ni/website/backend/controller/ProjectController.kt

```diff
@@ -9,11 +9,12 @@ import org.springframework.web.bind.annotation.RequestBody
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
 import pt.up.fe.ni.website.backend.dto.entity.ProjectDto
+import pt.up.fe.ni.website.backend.service.ActivityService
 import pt.up.fe.ni.website.backend.service.ProjectService
 
 @RestController
 @RequestMapping("/projects")
-class ProjectController(private val service: ProjectService) {
+class ProjectController(private val service: ProjectService, private val activityService: ActivityService) {
```

Comment:
This is a bit awkward. What do you think of making `ProjectService` extend `ActitvityService`?

Reply:
Tried to implement this on my latest commit, wasn't sure of a good way to tie in the `ActivityRepository` to `ProjectService` and `EventService`, let me know what you think

Reply:
The repository should not be tied to the service. I think this structure should be possible:
```
ProjectService(projectRepository) : Activity service(projectRepository)```
Since `ProjectRepository` is also an `ActivityRepository`.

Reply:
I haven't tested anything but this structure might work for what you're suggesting: https://stackoverflow.com/questions/43350397/spring-data-jpa-inheritance-in-repositories

Reply:
The inheritance between repositories was added in my latest commits, check it out!